src/rockstor/system/services.py

```diff
@@ -192,8 +147,15 @@ def service_status(service_name, config=None):
     elif service_name == "sftp":
         # Delegate sshd's sftp subsystem status check to system.ssh.py call.
         return is_sftp_running(return_boolean=False)
-    elif service_name in ("replication", "data-collector", "ztask-daemon"):
-        return superctl(service_name, "status", throw=False)
+    elif service_name == "scheduling":
+        out, err, rc = systemctl("rockstor-scheduling", "status")
+        return out, err, rc
+    elif service_name == "rockstor-collector":
+        out, err, rc = systemctl("rockstor-collector", "status")
+        return out, err, rc
+    elif service_name == "replication":
+        out, err, rc = systemctl("rockstor-replication", "status")
+        return out, err, rc
```

Comment:
```suggestion
    elif service_name == "scheduling":
        return init_service_op("rockstor-scheduling", "status", throw=False)
    elif service_name == "replication":
        return init_service_op("rockstor-replication", "status", throw=False)
```

@Hooverdan96 here we use the more appropriate init_service_op for scheduling and replication as they both need a DB to systemd service name switch. However rockstor-collector does not require this name switch and so will fall-thought to the same init_service_op() call at the end of this procedure; only using its DB (and systemd) name. This should fix this branches previously detailed failings re visiting SYSTEM-Services Web-UI page.